docs/basic-concepts.rst

```diff
@@ -68,15 +68,14 @@ Repositories
 Flatpak applications and runtimes are typically stored and published using
 repositories, which behave very similarly to Git repositories. A Flatpak
 repository can contain a single object or multiple objects, and each object
```

Comment:
I think `object`s here should be explained in simple terms a little bit because it is used almost nowhere else in the docs, which can cause some confusion. Maybe something like `, which somewhat resembles Git object,` is serviceable?

Reply:
This mostly belongs to ostree api docs not here. These docs are for end users and pacakgers who don't need to concern themselves on how the internal structure works.

The sentence before and this one should be joined together

Reply:
Something like

`Flatpak applications and runtimes are typically stored and published using Flatpak
repositories, which behave very like Git repositories allowing easy upgrades and downgrades`